app/src/androidMain/kotlin/de/westnordost/streetcomplete/quests/step_count/AddStepCountForm.kt

```diff
@@ -1,9 +1,9 @@
 package de.westnordost.streetcomplete.quests.step_count
 
 import de.westnordost.streetcomplete.R
-import de.westnordost.streetcomplete.quests.AddCountInput
-
-class AddStepCountForm : AddCountInput() {
+import de.westnordost.streetcomplete.quests.AAddCountInput
 
+class AddStepCountForm : AAddCountInput() {
```

Comment:
```suggestion
class AddStepCountForm : AddCountInput() {
```

Reply:
likely needs to be changed also elsewhere

Reply:
`A` is for abstract